app/_kong_plugins/datakit/examples/authenticate-third-party.yaml

```diff
@@ -0,0 +1,61 @@
+description: |
```

Comment:
With all the plugin examples, I kept getting a `datakit' is an enterprise only plugin` despite having an enterprise license configured.

Reply:
Resolved, we figured out it was a license issue. Everything works.